.github/drafts/16296/16296_self-hosting.md

```diff
@@ -0,0 +1,74 @@
+# Self-hosting
+Self-hosting is hosting and managing service directly by the owner on your own server rather than relying on a third-party service provider. Some examples include web hosting, file hosting, and email hosting. Self-hosting is often seen as a more secure and private alternative to using third-party services, as the user has complete control over their data and how it's used.
+
+Appsmith is a low-code, open source app development tool that offers a fast and easy way to develop apps. It provides a [cloud-hosted version](https://app.appsmith.com)[cloud-hosted] and the option to set up your own instance on some popular [platforms](# Deploy).
+
+## Cloud-hosted
+Cloud-hosted version is available over the internet, which makes it easy for users to sign up and start building apps. It's a perfect solution for users who don't have their own infrastructure setup. You can sign up on appsmith.com and follow the [Quick start guide](start-building) to get started.
+
+## Self-hosted
+You can choose to deploy Appsmith on your own infrastructure. It's often useful when you would want to plug it into existing infrastructure without the need to maintain a different setup. It enables you to take advantage of what your infrastructure has to offer and extend it to Appsmith. 
```

Comment:
You can add one line for explaining self-hosting here.

Reply:
This section talks about self-hosting for Appsmith. And expands on the thought that was added to the introduction. The self-hosting as a general concept should be at the start of the documentation. Let's discuss this if needed.

Reply:
We can skip this line "Appsmith is a low-code, open source app development tool that offers a fast and easy way to develop apps."  and directly mention - there are two ways to use the Appsmith platform.

Reply:
done.